test/create-snowpack-app/create-snowpack-app.test.js

```diff
@@ -88,7 +88,7 @@ describe('create-snowpack-app', () => {
     const cwd = path.join(TEMPLATES_DIR, template);
 
     it(`${template} > build`, async () => {
-      await execa('yarn', ['build', '--clean'], {
+      await execa('yarn', ['build', '--clean', '--baseUrl', '/'], {
```

Comment:
@markfirmware this should fix the snapshot tests (app-templates are strange because they are both a package and a half app template. We need a homepage for the npm search page, which you would never set for an application.